rust/src/io/reader.rs

```diff
@@ -177,6 +177,10 @@ impl<'a> FileReader<'a> {
         self.metadata.num_batches()
     }
 
+    pub fn batch_length(&self, batch_id: i32) -> usize {
+        return self.metadata.get_batch_length(batch_id).unwrap_or_default() as usize
```

Comment:
what's default here?

Reply:
Default is 0.  If happens if the batch_id is out range.